docker-compose.yml

```diff
@@ -1,14 +1,20 @@
 ---
-version: "3.7"
+x-orchestrator-config: &orchestrator-config
+  POSTGRES_USER: ${POSTGRES_USER:-postgres_user}
+  POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-postgres_pass}
+  POSTGRES_DB: ${POSTGRES_DB:-postgres_db}
+
+x-sot-config: &sot-config
+  INFRAHUB_TOKEN: ${INFRAHUB_TOKEN:-aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa}
```

Comment:
Better to leave the default one from Infrahub docs so `"06438eb2-8019-4776-878c-0941b1f1d1ec"`